invenio_deposit/api.py

```diff
@@ -211,18 +220,17 @@ def process_files(data):
 
             with process_files(data) as data:
                 record = Record.create(data, id_=id_)
+            self.commit()
```

Comment:
revert this change https://github.com/inveniosoftware/invenio-deposit/pull/63/files#r64379353